src/handlers/GraphQLHandler.test.ts

```diff
@@ -298,6 +306,15 @@ describe('predicate', () => {
     )
   })
 
+  test('respects graphql.operation endpoint', () => {
```

Comment:
This is a great test but I think it should be added to the `test/graphql-api/operation.test.ts` test suite. That way the intention is closer to the feature (`graphql.operation`) that causes the issue.

Reply:
This will also require restructuring the test. Look at the existing test cases in the suite to see how the GraphQL requests are performed (it'll be a browser context). Feel free to ask any questions here.